pages/quantum.js

```diff
@@ -53,139 +61,453 @@ import {
 
 export default function Quantum() {
   return (
-    <Container>
+    <div class="ComponentsContainer ">
```

Comment:
![image](https://github.com/catho/ssr-quantum-simulator/assets/3389749/3dcb86fe-e544-4a7c-b8d9-a8ac8bf7fd0c)

In react render, we dont use `class` but `className`.
Same for `datetime`, use `dateTime` instead

Look for this changes in another files of the project, because the alerts is showing in terminal from other places